context.go

```diff
@@ -240,6 +251,19 @@ type contextSettings struct {
 	JobIDResponse            *string
 }
 
+// loadContextResponseValue loads generic values from the response and puts it into variables specified
+// via context values.
+func loadContextResponseValues(cs contextSettings, resp Response) {
+	// Parse potential dirty read
+	if cs.AllowDirtyReads && cs.DirtyReadFlag != nil {
```

Comment:
Don't think you need to consider `AllowDirtyReads` here. `DirtyReadFlag != nil` is sufficient.